storage/client_raft_test.go

```diff
@@ -382,7 +389,7 @@ func TestFailedReplicaChange(t *testing.T) {
 
 	// The pending config change flag was cleared, so a subsequent attempt
 	// can succeed.
-	storage.TestingCommandFilter = nil
+	runFilter.Store(false)
```

Comment:
nah, only this change is only necessary when `TestingCommandFilter` is being turned off in the middle of a test (and causing a race)